trxdb-loader/loader.go

```diff
@@ -41,6 +42,8 @@ type TrxDBLoader struct {
 	batchSize                 uint64
 	lastTickBlock             uint64
 	lastTickTime              time.Time
+	lastBlockSecs             []float64
```

Comment:
maybe prevBlockRates would be clearer on the intent ? I had a hard time figuring out what it actually meant

then prevBlockRatesPointer